core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -59,6 +59,9 @@
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
 
+import static com.netflix.conductor.common.metadata.tasks.Task.Status.*;
+import static com.netflix.conductor.common.metadata.tasks.Task.Status.FAILED;
```

Comment:
This is already imported in the line above.

Reply:
Thank you, fixed my IDE settings.